_android/tutorial.md

```diff
@@ -3,7 +3,7 @@ layout: learn
 description: Learn about the Android SDK
 permalink: /:collection/:path.html
 ---
-# Android SDK Tutorial (Pre-release)
+# Android DApps (Pre-release)
```

Comment:
I think we can just remove "Pre-release".

Reply:
Done